.gitignore

```diff
@@ -1,15 +1,5 @@
 node_modules
 dist
 yarn-error.log
-/test/compiler/build/
-/test/projects/custom/custom_build
-/test/projects/insideOut/build
-/test/projects/custom_solidity_4/custom_build
-/test/projects/compilerOptions/build
-/test/projects/humanReadableAbi/build
-/test/matchers/build/
-/test/example/build/
-/build
-/tmp
```

Comment:
`.gitignore` for `waffe-provider` and `waffle-chai` seems to be missing

Reply:
It automatically inherits from parent .gitignore. So no need for them.